virttest/tests/unattended_install.py

```
@@ -1569,6 +1570,67 @@ def copy_images():
+                            "Domain restarted, updated serial console log from: %s to: %s",
+                            old_log_file,
+                            log_file,
+                        )
+
+                        # Note: Completion checking will be handled in the main monitoring loop
+
+                        LOG.debug(
+                            "Continuing installation monitoring with new log file"
+                        )
+                        continue  # Continue monitoring seamlessly
+                    except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)